nekoyume/Assets/_Scripts/UI/Combination/SubRecipeView.cs

```diff
@@ -561,6 +554,39 @@ private void UpdateButton()
             }
         }
 
+        private void UpdateButtonForConsumable()
+        {
+            var submittable = CheckNCGAndSlotIsEnough();
+            var inventory = States.Instance.CurrentAvatarState.inventory;
+            foreach (var material in _selectedRecipeInfo.Materials)
+            {
+                if (!TableSheets.Instance.MaterialItemSheet.TryGetValue(material.Key, out var row))
+                {
+                    continue;
+                }
+
+                var itemCount = inventory.TryGetFungibleItems(row.ItemId, out var outFungibleItems)
+                    ? outFungibleItems.Sum(e => e.count)
+                    : 0;
+
+                // when a material is unreplaceable.
+                if (material.Value > itemCount &&
+                    !TableSheets.Instance.CrystalMaterialCostSheet.ContainsKey(material.Key))
```

Comment:
```
                    !TableSheets.Instance.CrystalMaterialCostSheet.ContainsKey(material.Key))
```
It needs? As far as I know, food materials are always unreplaceable.

Reply:
I missed this.. I will delete that part and update comment. thank you!